bin/spring

```diff
@@ -7,9 +7,11 @@ unless defined?(Spring)
   require 'rubygems'
   require 'bundler'
 
-  if (match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m))
-    Gem.paths = { 'GEM_PATH' => [Bundler.bundle_path.to_s, *Gem.path].uniq }
-    gem 'spring', match[1]
+  lockfile = Bundler::LockfileParser.new(Bundler.default_lockfile.read)
```

Comment:
What's this?

Reply:
Just tried to pull down your branch to view the help pages.
Trying to run `bundle install` returns this error:
```
Your bundle is locked to parser (2.5.0.4), but that version could not be found in any of the sources listed in your Gemfile.
If you haven't changed sources, that means the author of parser (2.5.0.4) has removed it.
You'll need to update your bundle to a different version of parser (2.5.0.4) that hasn't been removed in order to install.
```

Reply:
Not sure what happened here but its resolved in my latest commit.

Reply:
Kind of related to my above comment, but if this is was an unintentional change then we shouldn't merge it into master.  I just reverted this file and the `Gemfile.lock` and everything still runs fine.
This should be reverted as well.